README.md

```diff
@@ -89,11 +89,9 @@ end
    ![Deployment Target](screenshots/deployment-target.png)
 10. Change "View controller-based status bar appearance" to `YES` in `Info.plist`:
     ![View Controller-Based Status Bar Appearance](screenshots/view-controller-based-status-bar-appearance.png)
-11. If your application is targeting iOS versions **prior to iOS 12.2** and your application **does not already contain any Swift code**, then you need to make sure Xcode bundles Swift standard libraries with your application distribution. To to so, open your target Build Settings and enable `Always Embed Swift Standard Libraries`:
-    ![Always Embed Swift Standard Libraries](screenshots/always-embed-swift-standard-libraries.png)
```

Comment:
This is no longer required in Xcode 12.2. The setting is on by default.

Reply:
Nice! I tested this as well, and saw the same.